gui/liquids.lua

```diff
@@ -178,6 +232,7 @@ end
 SpawnLiquidScreen = defclass(SpawnLiquidScreen, gui.ZScreen)
 SpawnLiquidScreen.ATTRS {
     focus_path='spawnliquid',
+    pass_movement_keys=true,
```

Comment:
You probably also want to set `pass_mouse_clicks=false` and `force_pause=true` since you're affecting the map